kitchen-sink-demo/src/main.ts

```diff
@@ -1,14 +1,31 @@
-import { enableProdMode } from '@angular/core';
+import { enableProdMode, NgZone } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+// @todo: how to not include in prod bundle?
+import { auguryBootstrap } from '../../../augury-labs/packages/core/dist'; // '@augury/core';
```

Comment:
I'm not a fan of using a relative import like this considering this is a completely different project it's using. It actually feels more like this sort of test might be better placed in augury-labs. At a minimal we should use the new delivery model then @santiago-elustondo and I came up with instead of this old approach.

This requires 3 file changes

```
main.augury.ts (new)
angular.json (update transform to use this on new build config option)
package.json (optional - add new serve option with new build config option)
```